movie/src/index.js

```diff
@@ -1,13 +1,13 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
-import App from './App';
+import Movie from './components/movie/Movie'
```

Comment:
This is bad.

`index.js` should render `app.js`. **ALWAYS**

So you can move the `Movie` into `App` and bring `App` back to this file